modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc

```diff
@@ -50,3 +50,14 @@ Microsoft Office Online access can be restricted to users in a specific group, b
 In the example above, only users in the `admin` group would be able to access Microsoft Office Online.
 
 NOTE: If the key is not defined, then all users have access to WOPI.
+
+== Locking the document
+
+If you open a document with Microsoft Office Online in ownCloud, then it makes use of the WebDAV file locking functionality, which we have in the server. 
+The idea is to lock the file so other users, who have access, can't make changes to the document, while you're editing it. 
+
+In other words: the feature ensures that you are the "master editor", your changes will always be the "master state". 
+Other users can make changes, e.g., with the desktop client, but those will create conflict files for them which can be resolved afterwards.
+When you close the document, Microsoft Office Office Online also has a feature to unlock the file again so it can be edited by others again as you are done with it.
```

Comment:
```suggestion
When you close the document, Microsoft Office Office Online will unlock the file so it can be edited by others.
```
I suggest a shorter sentence here. It is supposed to "just work".

Reply:
Is it "Microsoft Office Office Online" or just "Microsoft Office Online"?

Reply:
:D good catch :)

Reply:
fixed